core/site.go

```diff
@@ -443,6 +463,14 @@ func (site *Site) updateMeters() error {
 				site.log.ERROR.Printf("battery %d power: %v", i+1, err)
 			}
 
+			var energy float64
+			if m, ok := meter.(api.MeterEnergy); err == nil && ok {
+				energy, err := m.TotalEnergy()
+				if err == nil {
+					site.batteryEnergy += energy
```

Comment:
Macht eine Summe von Zählerständen überhaupt Sinn? Wozu? 
Warum brauchen wir das als Variable an der site (es hat keine weitere Steuerungsaufgabe)?

Reply:
Imho schon, da man ja z.B. die Gesamtproduktion seiner diversen PV-Anlagen pro Stunde/Tag/Monat/Jahr auswerten können will.
Sonst müsste man dafür zwingend die Einzelanlagen selbst addieren.

An der site braucht man das vielleicht nicht unbedingt. Hab mich da erstmal nur analog zu Power orientiert. War einfach.

Könnte mir aber auch gut vorstellen dass wir das intern z.B. für die savings verwenden können

Reply:
> Imho schon, da man ja z.B. die Gesamtproduktion seiner diversen PV-Anlagen pro Stunde/Tag/Monat/Jahr auswerten können will.

Das kann ich ja dann in Influx als sum machen. So einen virtuellen Gesamtzählerstand finde ich schon ungewöhnlich?

> Könnte mir aber auch gut vorstellen dass wir das intern z.B. für die savings verwenden können

Dann sollten wir es aber auch erst dann als Zustandsvariable einführen. Viele Variablen -> viel Unfug drin ;)